### PR TITLE
Jam guided about me

### PR DESCRIPTION
**Pull request by Jazy1618**
I am still trying to figure out these servlets, and why I get a 503 error on the devserver now, but this is what I'm testing so far.

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+@WebServlet("/about")
+public class AboutMeServlet extends HttpServlet{
+  
+ @Override
+ public void doGet(HttpServletRequest request, HttpServletResponse response)
+   throws IOException {
+  
+  response.getOutputStream().println("hello world");
+ }
+}
```
Comment (bandreasv) [inline src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: Add an empty line at the end of this file. It is convention to have that like there as many compilers use that to know it's the end of the file.

File: src/main/java/com/google/codeu/servlets/AboutMeServelet.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.google.codeu.servlets;
```
Comment (bandreasv) [inline src/main/java/com/google/codeu/servlets/AboutMeServelet.java]: Delete this file. Having multiple files with @WebServlet("/about") might be getting the program confused.

File: src/main/java/com/google/codeu/servlets/AboutMeServlet2.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package com.google.codeu.servlets;
```
Comment (bandreasv) [inline src/main/java/com/google/codeu/servlets/AboutMeServlet2.java]: Delete this file. Having multiple files with @WebServlet("/about") might be getting the program confused.

**Review by bandreasv — CHANGES_REQUESTED** [review]

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java (hunk 1 above)
Comment (Jazy1618) [reply src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: So, there should be an empty line #23?

File: src/main/java/com/google/codeu/servlets/AboutMeServlet2.java (hunk 3 above)
Comment (Jazy1618) [reply src/main/java/com/google/codeu/servlets/AboutMeServlet2.java]: Ok

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java (hunk 1 above)
Comment (bandreasv) [reply src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: Yes

File: src/main/java/com/google/codeu/servlets/AboutMeServlet2.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import java.io.IOException;
```
Comment (bandreasv) [inline src/main/java/com/google/codeu/servlets/AboutMeServlet2.java:1]: Should also delete this file. You don't need more than one.

**Review by bandreasv — CHANGES_REQUESTED** [review]

**Review by bandreasv — APPROVED** [review]

**Pull request merged**